integration-tests/tests/test_eth_api.py

```diff
@@ -0,0 +1,55 @@
+import random
+import pytest
+import jsonschema
+import json
+from conftest import option
+from test_cases import EthApiTestCase
+
+
+@pytest.mark.rpc
+@pytest.mark.ethclient
+class TestRpc(EthApiTestCase):
+    @pytest.mark.parametrize(
+        "iterations", range(3) # Run multiple times, each with a different Tx
+    )
+
+    def test_(self, iterations):
+        _id = str(random.randint(1, 8888))
```

Comment:
I'm a python noob, why underscore? 🤔 
```suggestion
        id = str(random.randint(1, 8888))
```

Reply:
~I copy-pasted~ I followed the repo's coding standards
@antdanchenko any good reason why those start with an underscore?

Reply:
that is incapsulation pattern it seems, private var

Reply:
id is build in python var, I just didn't want to rewrite it

Reply:
`requestId` then? 🙂